widgets/layout-slider/layout-slider.php

```diff
@@ -192,7 +192,7 @@ function get_widget_form(){
 	}
 
 	function form( $instance, $form_type = 'widget' ) {
-		if( is_admin() && defined('SITEORIGIN_PANELS_VERSION') ) {
```

Comment:
This was added to prevent outputting the _Open Builder_ button in some contexts where it wouldn't work. Specifically it was in the front-end editor of some page builders, which lacked some of the scripts and styles required for the builder field to work.

Reply:
Ah. I wasn't aware of that. What would you recommend using to detect Gutenberg? I ask as I had a look around and I can't find anywhere, where we're specifically checking for Gutenberg, is active on the current page.

Reply:
As far as I can tell at the moment it's necessary to use two checks to stay compatible with both the Gutenberg plugin block editor and the integrated block editor in WP 5. See:
https://github.com/siteorigin/siteorigin-panels/blob/82627ae22b38f5ce831ca36d77de67b54e2b06ff/compat/gutenberg-block.php#L32-L38

Reply:
Annoyingly, that doesn't work as `is_current_screen` doesn't exist at the time. For whatever reason, it looks like WordPress hates conditionals now - a simple `is_guttenberg()` would be really convenient right about now. :/

Reply:
Ah, right. For a moment I forgot this is in the block editor context. 😕